events.qmd

```diff
@@ -12,16 +12,16 @@ title: Events
 #### Code for Philly Takeover and Pizza
 *Date: March 14, 2024*<br>
 Speaking: Caitlin Sherman, Kat Jost<br>
-Registration coming soon
+[Register now] (https://www.eventbrite.com/e/sjhphl-social-justice-projects-with-code-for-philly-pizza-nite-tickets-858595481197)
 
 #### Open Source Technology in Legal Aid and Pizza
 *Date: March 21, 2024*<br>
 Speaking: Kevin Mulcahy, Travis Southard, Jonathan Pyle<br>
-Registration coming soon
+[Register now] (https://www.eventbrite.com/e/860575954847)
 
 #### Open Data for Legal Aid Projects
 *Date: March 28, 2024*<br>
-Registration coming soon
+[Register now] (https://www.eventbrite.com/e/sjhphl-open-data-for-legal-aid-projects-and-pizza-tickets-860772532817)
```

Comment:
For the links to render correctly, there should be no spaces between the square brackets and the parentheses for the links. For example [this link] (http://example.com) will not render but [this one](http://example.com) will. If you can fix that I will approve these

From the deployment:
<img width="882" alt="image" src="https://github.com/socialjusticehackathon/website/assets/64238570/1bfed165-9a99-43c3-986a-59f94144ed26">